examples/reset-camera-example.py

```diff
@@ -16,8 +16,7 @@
 # -----------------------------------------------------------------------------
 
 
-@TrameApp()
-class ConeApp:
+class ConeApp(TrameApp):
```

Comment:
When you use the TrameApp base class, you need to do a couple things:
- import it from the right spot (trame.app vs trame.decorators)
- call `super().__init__(server, client_type="vue3")` and remove all the thing the class is doing for you. [see](https://kitware.github.io/trame/guide/versions/features.html#version-3-9) and that gives you access to self.state/ctrl/ctx so no need to create those props.